source/nodejs/adaptivecards-designer-app/dist/index.html

```diff
@@ -3,10 +3,10 @@
 
 <head>
     <title>Adaptive Cards Designer</title>    
-</head>
+<script defer src="adaptivecards-designer-app.js"></script></head>
```

Comment:
NIT: formatting of tags

Reply:
So TBH, this file seems to be auto-generated, yet still committed to the repo.  Not sure the reasoning behind it, but I just didn't want to mess with that lol.

Reply:
Note that it is in the `dist` directory.  I would like to remove it from the committed files, but given that it's already in there....not sure what to do lol

Reply:
Marking this as resolved as this is a pre-existing idiosyncrasy